src/index.tsx

```diff
@@ -1,19 +1,35 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-
 import * as Sentry from '@sentry/react';
 import { BrowserTracing } from '@sentry/tracing';
-import { CaptureConsole as CaptureConsoleIntegration } from '@sentry/integrations';
 
 import App from './App';
 
 Sentry.init({
   dsn: process.env.SENTRY_DSN,
   integrations: [
-    new BrowserTracing(),
-    new CaptureConsoleIntegration({ levels: ['error'] }),
+    new BrowserTracing()
   ],
   tracesSampleRate: 1.0,
+  beforeSend(event, hint) {
+    const error = hint.originalException;
+
+    const ignoreErrors = [
+      /GraphQL/i, // filter out graphql errors
```

Comment:
Wouldn't it also filter out real exceptions if they contain the GraphQL string?
This question doesn't block merge, but will need to be addressed in the future